functions/package.json

```diff
@@ -2,35 +2,38 @@
   "name": "functions",
   "private": true,
   "main": "lib/index.js",
+  "type": "module",
   "scripts": {
-    "build": "tsc",
+    "clean": "rimraf lib",
+    "build": "npm run clean && tsc",
     "build:watch": "tsc --watch",
     "check": "tsc -b",
     "deploy": "firebase deploy --only functions",
-    "lint": "eslint --ext .js,.ts .",
+    "lint": "eslint .",
```

Comment:
Do you need this here if you are linting at the root?

Reply:
This gets run prior to a deploy so I think it makes sense.